packages/grafana-e2e/src/flows/openPanelMenuItem.ts

```diff
@@ -3,10 +3,23 @@ import { e2e } from '../index';
 export enum PanelMenuItems {
   Edit = 'Edit',
   Inspect = 'Inspect',
+  More = 'More...',
```

Comment:
In a follow up PR we will move the UI extensions from the "More..." sub menu to its own "Extensions" sub menu.